src/openrct2/peep/peep.c

```diff
@@ -11394,6 +11406,9 @@ static bool peep_find_ride_to_look_at(rct_peep *peep, uint8 edge, uint8 *rideToV
 
     mapElement = surfaceElement;
     do {
+        if (network_get_mode() != NETWORK_MODE_NONE) {
```

Comment:
Please put a comment explaining this is to prevent desyncs from interaction with ghost elements.